DOWNLOADS.md

```diff
@@ -50,6 +50,8 @@ matches corresponding changes in the artifacts task.
   and
   [darwin_arm64](https://api.cirrus-ci.com/v1/artifact/github/containers/podman/Artifacts/binary/podman-remote-release-darwin_arm64.zip).
   if you try to install them.
+* FreeBSD [podman] builds for x86_64 only:
+  * [podman-release-freebsd-amd64.tar.gz](https://api.cirrus-ci.com/v1/artifact/github/containers/podman/Artifacts/binary/podman-release-freebsd-amd64.tar.gz)
```

Comment:
The code that generates this isn't working.  Please don't include this if people who click it are only going to get errors.

Reply:
done